dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/FATSuite.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.mcp.internal.fat;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import io.openliberty.mcp.internal.fat.lifecycle.tests.BeanLifecycleTest;
+import io.openliberty.mcp.internal.fat.lifecycle.tests.LifecycleTest;
+import io.openliberty.mcp.internal.fat.protocol.HttpTest;
+import io.openliberty.mcp.internal.fat.protocol.ProtocolVersionTest;
+import io.openliberty.mcp.internal.fat.tool.ToolTest;
+
+/**
+ *
+ */
+@RunWith(Suite.class)
+@SuiteClasses({
+                BeanLifecycleTest.class,
+                HttpTest.class,
+                LifecycleTest.class,
+                ProtocolVersionTest.class,
+                ToolTest.class
+
+})
+public class FATSuite {
```

Comment:
Do we need to do repeats for EE 10 and 11?  Not sure if repeat rules handle `platform` yet for versionless features.

Reply:
We will eventually, but not yet